.eslintrc.js

```diff
@@ -149,6 +149,8 @@ module.exports = {
     'rulesdir/prettier-comments': 'error',
     // Enforces consistent file extension
     'rulesdir/extensions': 'error',
+    // Enforces license headers on files
+    'rulesdir/check-license': 'error',
```

Comment:
I think the current version is already useful so if you want @richestrinh we could flip this to "warn" and land this as the first version